hack/util.sh

```diff
@@ -47,16 +47,34 @@ function configure_os_server() {
 		export SERVER_HOSTNAME_LIST
 	fi
 
+        CREATE_MASTER_CERTS="openshift admin ca create-master-certs"
+        CREATE_NODE_CONFIG="openshift admin create-node-config"
+        CREATE_BOOTSTRAP_POLICY="openshift admin create-bootstrap-policy-file"
+        CREATE_CONFIG="openshift start"
+
+        CONFIG_VERSION=${1:-}
+        if [[ -n "${CONFIG_VERSION}" ]]; then
```

Comment:
The indentation here is inconsistent with the rest of this file -- but I think since we're adding so many new functions and they're all grouped so tightly, it makes more sense to break this stuff out into `hack/lib/start.sh` and fix the indentation there instead of here.